incubator/java-openliberty/image/project/invoke-appsody.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [ -e /project/user-app/.appsody-nolocal ]
+then
+	BIN_TEMPLATE=1
+fi
+
+case $APPSODY_DEV_MODE in
+	run)
+		#if [ -e /project/user-app/.appsody-nolocal ]
+		if [ $BIN_TEMPLATE = 1 ]
+		then
+			echo this appsody run not supported when using binary template. Press ctl-C and/or stop the current container.
```

Comment:
Nit picking... there's more than one way to do this but I don't recall seeing `ctl-C`.  Could we maybe use `Ctrl+C` or something more common.

Reply:
eliminated this part of the message by moving this all to validate.sh